devicetypes/smartthings/zwave-panic-button.src/zwave-panic-button.groovy

```diff
@@ -45,16 +45,17 @@ metadata {
 
 def installed() {
 	log.debug "Installed $device.displayName"
-	sendEvent(name: "DeviceWatch-Enroll", value: [protocol: "zwave", scheme:"untracked"].encodeAsJson(), displayed: false)
+	//sendEvent(name: "DeviceWatch-Enroll", value: [protocol: "zwave", scheme:"untracked"].encodeAsJson(), displayed: false)
 	if(hasMultipleButtons()) {
 		addChildButtons()
 	}
-	setPanicAlarm("clear")
 	def cmds = [
 			zwave.batteryV1.batteryGet().format(),
 			zwave.notificationV3.notificationGet().format(),
+			"delay 1000",
 			zwave.wakeUpV1.wakeUpNoMoreInformation().format()
 	]
+	sendEvent(name: "panicAlarm", value: "clear", descriptionText: "Panic Alarm status: clear", displayed: true, isStateChange: true)
```

Comment:
was the notification report not taking care of this?

Reply:
nope, device doesn't answer to notificationGet at all. I should probably delete this command

Reply:
please do. other than that it looks fine